src/software/geom/rectangle.h

```diff
@@ -24,26 +24,26 @@ class Rectangle final
      *
      * @param neg_x_neg_y_corner The <-x,-y> corner of the rectangle
      *
-     * @param width The width of the rectangle
+     * @param xLength The xLength of the rectangle
      *
-     * @param height The height of the rectangle
+     * @param yLength The yLength of the rectangle
      */
-    explicit constexpr Rectangle(const Point &neg_x_neg_y_corner, double width,
-                                 double height);
+    explicit constexpr Rectangle(const Point &sw_corner, double x_length, double y_length);
```

Comment:
The parameter names should stay `neg_x_neg_y_corner` etc as before, we don't want to use "north east south west"